client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -210,6 +211,10 @@ public class ClientOptions
     @Option(names = "--schema", paramLabel = "<schema>", description = "Default schema")
     public Optional<String> schema;
 
+    @PropertyMapping(SQL_PATH)
+    @Option(names = "--path", paramLabel = "<schema>", description = "Default SQL path")
```

Comment:
`SET PATH` SQL query supports passing multiple values, should we support it here too?

Reply:
And then it is searched in order?

Reply:
Indeed - nice catch

Reply:
I've fixed the API in different places.